streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorRecordContext.java

```diff
@@ -79,17 +84,17 @@ public Headers headers() {
         return headers;
     }
 
-    public long sizeBytes() {
-        long size = 0L;
-        size += 8; // value.context.timestamp
-        size += 8; // value.context.offset
+    public int sizeBytes() {
```

Comment:
`long` is an unnecessarily large data type for `size`, since arrays can only be indexed with integers (thus, a byte array cannot hold more than a "max int" number of bytes).

Reply:
Actually, the latter conversation with @mjsax about the relationship (or lack of one) between this and the serialized form makes me think maybe this response was incorrect. I'll move it back to long.